rope/contrib/codeassist.py

```diff
@@ -13,12 +13,14 @@
 from rope.base import pyobjectsdef
 from rope.base import pyscopes
 from rope.base import worder
+from rope.base import utils
 from rope.contrib import fixsyntax
 from rope.refactor import functionutils
 
 
 def code_assist(project, source_code, offset, resource=None,
-                templates=None, maxfixes=1, later_locals=True):
+                templates=None, maxfixes=1, later_locals=True,
+                case_sensitive=False):
```

Comment:
For backward compatibility I think a default value of case_sensitive should be a True.